pkg/sensors/tracing/syscall_list.go

```diff
@@ -51,6 +51,8 @@ func (v SyscallVal) Symbol() (string, error) {
 		prefix = "__arm64_"
 	case "i386":
 		prefix = "__ia32_"
+	case "arm32":
+		prefix = "__arm64_"
```

Comment:
Could we add a comment here explaining the same thing you explained in the commit message about the `__arm64_` prefix?

Reply:
God point, added a comment.